src/app/core/json-patch/json-patch-operations.reducer.ts

```diff
@@ -0,0 +1,302 @@
+import { hasValue, isNotEmpty, isNotUndefined, isNull } from '../../shared/empty.util';
+
+import {
+  FlushPatchOperationsAction,
+  PatchOperationsActions,
+  JsonPatchOperationsActionTypes,
+  NewPatchAddOperationAction,
+  NewPatchCopyOperationAction,
+  NewPatchMoveOperationAction,
+  NewPatchRemoveOperationAction,
+  NewPatchReplaceOperationAction,
+  CommitPatchOperationsAction,
+  StartTransactionPatchOperationsAction,
+  RollbacktPatchOperationsAction
+} from './json-patch-operations.actions';
+import { JsonPatchOperationModel, JsonPatchOperationType } from './json-patch.model';
+
+export interface JsonPatchOperationObject {
```

Comment:
Please add TypeDoc/comments to describe this new interface (and others in this same file).  I see you only added comments for the "JsonPatchOperationsState" interface, but all interfaces should have similar descriptions.